kansha/board/boardsmanager.py

```diff
@@ -59,15 +60,23 @@ def _get_board(self, data_board):
         return self._services(Board, data_board.id, self.app_title, self.app_banner, self.theme,
                               self.card_extensions, self.search_engine)
 
+    def create_template_empty(self):
+        '''Get a default empty template'''
+        board = DataBoard(title=u'Empty board', is_template=True)
+        board.visibility = BOARD_PUBLIC
```

Comment:
Could it be possible to derivate `TemplateBoard` from `Board`? So `is_template` and the default `visibility` could be set in `TemplateBoard`